pymc3/tests/test_distributions.py

```diff
@@ -989,6 +971,27 @@ def test_wald(self, value, mu, lam, phi, alpha, logp):
         decimals = select_by_precision(float64=6, float32=1)
         assert_almost_equal(model.fastlogp(pt), logp, decimal=decimals, err_msg=str(pt))
 
+    def test_wald_logp(self):
+        self.check_logp(
+            Wald,
+            Rplus,
+            {"mu": Rplus, "alpha": Rplus},
+            lambda value, mu, alpha: sp.invgauss.logpdf(value, mu=mu, loc=alpha),
+            decimal=select_by_precision(float64=6, float32=1),
```

Comment:
Here too - 1 digit precision?!

Reply:
This preceeded this PR as well

Reply:
By the way these 1 precision comparisons are not necessarily as bad as it may sound. It's usually when the log becomes very negative, close to underflowing to -inf things like -213.161 vs -213.171 that mean basically nothing